### PR TITLE
added button, yml file, dropdown format

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
About This PR:
- Added `Contribute with Gitpod` Button
- Make Code Snippets a dropdown to make it more accessible.
- Added `.gitpod.yml` template file

### CODE REVIEW

File: app/routes/index.tsx [hunk 1]
```diff
@@ -134,28 +137,94 @@ export default function Index() {
                 alt="Open in Gitpod"
                 width="200"
                 height="56"
-                src="/open-in-gitpod.svg"
+                src="https://gitpod.io/button/open-in-gitpod.svg"
```
Comment (jacobparis) [inline app/routes/index.tsx]: I’d prefer to leave this self hosted, it saves a few hundred ms on the image load for cache misses
```suggestion
                src="/open-in-gitpod.svg"
```

**Review by jacobparis — APPROVED** [review]
92083B07-B94B-4EAE-AB3A-F8205E4ABDD3

this was the first pr I opened in Gitpod on my phone, gonna have to get my mobile settings cleaned up

One comment is that on slow connections the buttons appear a split second later on the page when they’re being referenced externally, so I self hosted the first button when it’s rendered on page

the code snippets should still point to the one on Gitpod servers

Otherwise this PR looks great! It’s approved and I’ll merge when I get home

File: app/routes/index.tsx (hunk 1 above)
Comment (Siddhant-K-code) [reply app/routes/index.tsx]: Ok, i will upload that file (for contribute with gitpod svg ) and update this line in next commit.

Comment (Siddhant-K-code) [reply app/routes/index.tsx]: resolved in [`d0d5120`](https://github.com/jacobparis/gitpodify/pull/2/commits/d0d512008b4b7045faef18c1661b75e66c2fcf88)

**Pull request merged**